src/main/java/ch/heigvd/digitalpatate/monopoly/MGame.java

```diff
@@ -27,16 +27,28 @@ public MGame(int playerCount) throws IllegalArgumentException {
 
         players = new Player[playerCount];
         for (int i = 0; i < players.length; i++) {
-            players[i] = new Player("Player" + (i + 1), dices, board);
+            players[i] = new Player("Player " + (i + 1), dices, board);
         }
     }
 
     public void playGame() {
 
+        while (roundCount<20){
```

Comment:
Could be nice to have a constant or variable instead of the magic number.

Reply:
On it.